tests/kafkatest/services/kafka/kafka.py

```diff
@@ -94,7 +95,6 @@ def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAI
         self.zk_set_acl = False
         self.server_prop_overides = server_prop_overides
         self.log_level = "DEBUG"
-        self.num_nodes = num_nodes
```

Comment:
For my benefit: what is the reason for removing this? I also don't understand the related change in this file.